Countly.js

```diff
@@ -391,6 +396,38 @@ Countly.getCurrentDeviceId = async function () {
     return result;
 };
 
+_getDeviceIdType = function (deviceIdType) {
```

Comment:
let's do explicit cases.
default result value is null
in the default case we also print an error

Reply:
print the unexpected value also